x-pack/plugins/security_solution/public/helpers.ts

```diff
@@ -115,3 +140,10 @@ export const getInspectResponse = <T extends FactoryQueryTypes>(
   response:
     response != null ? [JSON.stringify(response.rawResponse, null, 2)] : prevResponse?.response,
 });
+
+export const isDetectionsPath = (pathname: string): boolean => {
+  return !!matchPath(pathname, {
+    path: `(${ALERTS_PATH}|${RULES_PATH}|${EXCEPTIONS_PATH})`,
```

Comment:
I think we want details rules path

Reply:
I can not adapt it to the sourcerer here, it is used by other parts. I will move the sourcerer "detections" route match to another function. Do we need it in this PR?

Reply:
I tested the code locally and everything is working as expected but I thought it will be cleaner to make it more precise.